packages/buefy-next/src/components/tabs/TabItem.spec.js

```diff
@@ -74,6 +74,24 @@ describe('BTabItem', () => {
         expect(wrapper.vm.transitionName).toBe('slide-next')
     })
 
+    it('should update active state', async () => {
+        const wrapperFirstItem = wrapperParent.findComponent({ ref: 'firstItem' })
+        // we need `<a>` elements corresponding to individual tabs
```

Comment:
We may need them but why do we need them?

Reply:
I will change it to:
```js
        // we need `<a>` elements corresponding to individual tab components to
        // activate them with 'click'
```
Does it make sense?